packages/firebase_crashlytics/firebase_crashlytics_platform_interface/test/mock.dart

```diff
@@ -2,17 +2,15 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// @dart=2.9
-
 import 'package:firebase_core_platform_interface/firebase_core_platform_interface.dart';
 import 'package:flutter/services.dart';
 import 'package:flutter_test/flutter_test.dart';
 import 'package:firebase_crashlytics_platform_interface/src/method_channel/method_channel_crashlytics.dart';
 
 typedef MethodCallCallback = dynamic Function(MethodCall methodCall);
-typedef Callback(MethodCall call);
+typedef Callback = void Function(MethodCall call);
 
-setupFirebaseCrashlyticsMocks([Callback /*?*/ customHandlers]) {
+dynamic setupFirebaseCrashlyticsMocks([Callback? customHandlers]) {
```

Comment:
```suggestion
void setupFirebaseCrashlyticsMocks([Callback? customHandlers]) {
```

Shouldn't this be void, nothing is returned here?